bears/general/AnnotationBear.py

```diff
@@ -88,66 +95,130 @@ def find_annotation_ranges(self,
+                                                          start_delim,
+                                                          end_delim,
+                                                          position,
+                                                          end_position)
+                    multiline_comment = {}
+                    multiline_comment[
+                        'start_delimiter_range'] = seperate_ranges[0]
+                    multiline_comment[
+                        'end_delimiter_range'] = seperate_ranges[1]
+                    multiline_comment['content_range'] = seperate_ranges[2]
+                    multiline_comment['full_range'] = seperate_ranges[3]
+                    multiline_comment_range.append(multiline_comment)
                     return end_position + 1
 
-                _range, end_position = self.get_range_end_position(
-                    file,
-                    filename,
-                    text,
-                    comment_delimiter,
-                    position,
-                    self.get_singleline_comment,
-                    single_comment=True)
-                if end_position and _range:
-                    comments_range.append(_range)
+                end_position, start_delim, end_delim = \
+                    self.get_range_end_position(
+                        file,
+                        filename,
+                        text,
+                        comment_delimiter,
+                        position,
+                        self.get_singleline_comment,
+                        single_comment=True)
+                if end_position:
+                    seperate_ranges = get_seperate_ranges(file,
+                                                          filename,
+                                                          start_delim,
+                                                          end_delim,
+                                                          position,
+                                                          end_position)
+                    singleline_comment = {}
+                    singleline_comment[
+                        'start_delimiter_range'] = seperate_ranges[0]
+                    singleline_comment[
+                        'end_delimiter_range'] = seperate_ranges[1]
+                    singleline_comment['content_range'] = seperate_ranges[2]
+                    singleline_comment['full_range'] = seperate_ranges[3]
+                    singleline_comment_range.append(singleline_comment)
                     return end_position + 1
 
                 return position + 1
 
             position = get_new_position()
 
-        return tuple(strings_range), tuple(comments_range)
+        return (tuple(singleline_string_range),
+                tuple(multiline_string_range),
+                tuple(singleline_comment_range),
+                tuple(multiline_comment_range))
```

Comment:
Could you add a comment on the PR, not linked to a commit, that explains why you are using tuples instead of named tuples, and link to the discussions involved in the decision.